modular_skyrat/modules/ashwalkers/Ashwalker/Ashwalkers.dm

```diff
@@ -0,0 +1,3 @@
+/datum/species/lizard/ashwalker
+	mutanteyes = /obj/item/organ/eyes/night_vision
+	weather_immunities = list("ash")
```

Comment:
You dare come into these lands without the proper newlines? Begone thot!

Reply:
A